src/components/PasswordResetFlow/ResetPage/index.js

```diff
@@ -19,111 +16,48 @@ const ResetPage = () => {
     })
   }
 
-  const openModal = () => {
-    setModalIsOpen(true)
-  }
-
-  const closeModal = () => {
-    setModalIsOpen(false)
-  }
-  console.log(errors)
-
   return (
-    <main role="main" className="password-reset-container">
+    <div className="password-reset-container">
       <h1 className="main-title">Kohdataan</h1>
 
       <div className="password-reset-content-container">
-        <h2 className="password-reset-title">Salasanan vaihtaminen</h2>
-        <div className="formfield-container">
-          <div className="info-circle-line">
-            <ValidatedInputField
-              label="Salasana"
-              name="password"
-              ref={register({
-                required: true,
-                maxLength: 30,
-                // password must contain lower and upper case letters and numbers
-                pattern: /^(?=.*[0-9]+.*)(?=.*[a-zäöüß]+.*)(?=.*[A-ZÄÖÜ]+.*)[\w\W]{10,}$/,
-              })}
-              type="password"
-              ariaInvalid={!!errors.password}
-              inputClassName="create-account-input-text"
-              labelClassName={
-                errors.password
-                  ? 'create-account-errors-field'
-                  : 'create-account-input-field'
-              }
-            />
-            <div className="info-circle">
-              <button
-                type="button"
-                onClick={openModal}
-                className="info-circle-button"
-                aria-labelledby="password-info"
-              />
-            </div>
-            <ToolTipModalContainer
-              modalIsOpen={modalIsOpen}
-              closeModal={closeModal}
-              label="show-password-info-dialog"
-              content="Salasanassa tulee olla vähintään 10 merkkiä, ja siinä pitää
-                olla isoja kirjaimia, pieniä kirjaimia ja numeroita."
-            />
-          </div>
-          <div className="error-text">
-            {errors.password &&
-              errors.password.type === 'required' &&
-              'Kirjoita salasana'}
-            {errors.password &&
-              errors.password.type === 'pattern' &&
-              'Salasanan on oltava vähintään 10 merkkiä pitkä ja siinä pitää olla isoja kirjaimia, pieniä kirjaimia ja numeroita.'}
-            {errors.password &&
-              errors.password.type === 'maxLength' &&
-              'Salasanan on oltava enintään 30 merkkiä pitkä.'}
-          </div>
-        </div>
+        <h2>Salasanan vaihtaminen</h2>
```

Comment:
According to Zeplin this should be "Salasanan palauttaminen". Probably my bad.

Reply:
Fix'd :)